File: FJT.ReportViewer/MySqlDBModel/reportmasterparameter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FJT.ReportViewer.MySqlDBModel
{
    public class reportmasterparameter
    {
        [Key]
        public int id { get; set; }

        public int reportId { get; set; }

        public int parmeterMappingid { get; set; }

        public bool isRequired { get; set; }

        public bool isDeleted { get; set; }

        [StringLength(255)]
        public string createdBy { get; set; }

        [StringLength(255)]
        public string updatedBy { get; set; }

        [StringLength(255)]
        public string deletedBy { get; set; }

        public DateTime createdAt { get; set; }

        public DateTime? updatedAt { get; set; }

        public DateTime? deletedAt { get; set; }

        public int createByRoleId { get; set; }

        public int? updateByRoleId { get; set; }

        public int? deleteByRoleId { get; set; }

        [ForeignKey("reportId")]
        public virtual reportmaster reportmaster { get; set; }

        [ForeignKey("parmeterMappingid")]
        public virtual report_parameter_setting_mapping Report_Parameter_Setting_Mapping { get; set; }
    }
}

File: fjt.api/models/ReportMasterParameter.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const ReportMasterParameter = sequelize.define('ReportMasterParameter', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        reportId: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        parmeterMappingid: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        isRequired: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        deletedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        tableName: 'reportmasterparameter',
        paranoid: true
    });
    ReportMasterParameter.associate = (models) => {
        ReportMasterParameter.belongsTo(models.ReportMaster, {
            foreignKey: 'reportId',
            as: 'ReportMaster'
        });
        ReportMasterParameter.belongsTo(models.ReportParameterSettingMapping, {
            foreignKey: 'parmeterMappingid',
            as: 'ReportParameterSettingMapping'
        });
    };
    return ReportMasterParameter;
};
